src/components/asset-list/RecyclerAssetList/index.tsx

```diff
@@ -153,19 +153,21 @@ const isEqualDataProvider = new DataProvider((r1, r2) => {
   }
 });
 
-const StyledRecyclerListView = styled(RecyclerListView)`
-  background-color: ${({ theme: { colors } }) => colors.white};
-  display: flex;
-  flex: 1;
-  min-height: 1;
-`;
-
-const StyledContainer = styled(View)`
-  display: flex;
-  flex: 1;
-  background-color: ${({ theme: { colors } }) => colors.white};
-  overflow: hidden;
-`;
+const StyledRecyclerListView = styled(RecyclerListView)({
+  // @ts-ignore
```

Comment:
We should use `@ts-expect-error` so these get cleaned up if the types are fixed.

Reply:
Yeah it makes sense, thanks